test/ctia/entity/web_test.clj

```diff
@@ -0,0 +1,525 @@
+                   {status :status
+                    judgement :parsed-body
+                    :as response}
+                   (post "ctia/judgement"
+                         :body {:observable {:value "1.2.3.4"
+                                             :type "ip"}
+                                :external_ids ["http://ex.tld/ctia/judgement/judgement-123"
+                                               "http://ex.tld/ctia/judgement/judgement-456"]
+                                :id "http://localhost:3001/ctia/judgement/judgement-00001111-0000-1111-2222-000011112222"
+                                :disposition 2
+                                :source "test"
+                                :priority 100
+                                :severity "High"
+                                :confidence "Low"
+                                :reason "This is a bad IP address that talked to some evil servers"
+                                :valid_time {:start_time "2016-02-11T00:40:48.212-00:00"}}
```

Comment:
`:body new-judgement-1` :-)

Reply:
not this one, there is an `id` field added.